indico/modules/events/contributions/forms.py

```diff
@@ -105,7 +105,9 @@ def __init__(self, *args, **kwargs):
         super(ContributionProtectionForm, self).__init__(*args, **kwargs)
 
     def validate_permissions(self, field):
-        check_permissions(self.event, field)
```

Comment:
oops :D thanks for fixing that

Reply:
I fixed something without knowing :sweat_smile: